src/ResourceManagement/DataLake.Analytics/Microsoft.Azure.Management.DataLake.Analytics/Generated/CatalogOperations.cs

```diff
@@ -1051,7 +1433,7 @@ internal CatalogOperations(DataLakeAnalyticsCatalogManagementClient client)
         /// <return>
         /// A response object containing the response body and response headers.
         /// </return>
-        public async Task<AzureOperationResponse<USqlExternalDataSource>> GetExternalDataSourceWithHttpMessagesAsync(string accountName, string databaseName, string externalDataSourceName, Dictionary<string, List<string>> customHeaders = null, CancellationToken cancellationToken = default(CancellationToken))
+        public async Task<AzureOperationResponse<USqlCredential>> GetCredentialWithHttpMessagesAsync(string accountName, string databaseName, string credentialName, Dictionary<string, List<string>> customHeaders = null, CancellationToken cancellationToken = default(CancellationToken))
```

Comment:
ignore this, looks like [https://github.com/Azure/azure-sdk-for-net/pull/2289/](https://github.com/Azure/azure-sdk-for-net/pull/2289/) already has the major/minor version updates.

---
In reply to: [77389322](https://github.com/Azure/azure-sdk-for-net/pull/2315#discussion_r77389322) [](ancestors = 77389322)